port/win32/omrsysinfo.c

```diff
@@ -1576,3 +1576,38 @@ omrsysinfo_get_open_file_count(struct OMRPortLibrary *portLibrary, uint64_t *cou
 	return OMRPORT_ERROR_SYSINFO_GET_OPEN_FILES_NOT_SUPPORTED;
 }
 
+intptr_t
```

Comment:
-- Moved changes in `win32/omrsysinfo.c` to `common/omrsysinfo.c`
-- Removed `omrsysinfo_os_has_feature`'s definition in `unix/omrsysinfo.c` - `omrsysinfo_os_has_feature`'s definition in `common/omrsysinfo.c` can be used by all platforms